### PR TITLE
Tidy up the readme, pushing working examples and changelog into the docs

### PR DESCRIPTION
**Pull request by joerick**
The README has been getting pretty long, especially with the changelog and working examples section. This change makes those sections generated by scripts - the latest 5 releases and top 10 projects - while keeping the full list in the docs. This should also fix the issue with dependabot not finding the release notes for each release  !

### CODE REVIEW

**Comment by joerick** [comment]
Restarted the GHA builds because the emulation tests had hung and were cancelled.

**Comment by joerick** [comment]
A @henryiii review on this would be great, if you have time? No worries if not!

File: bin/projects.py [hunk 1]
```diff
@@ -139,28 +139,38 @@ def fetch_icon(icon_name: str) -> None:
...
         for icon in ICONS:
             fetch_icon(icon)
 
     github = Github(auth) if online else None
 
-    projects = sorted((Project(item, github) for item in config), reverse=online)
+    return sorted((Project(item, github) for item in config), reverse=online)
+
+
+def render_projects(projects: list[Project], *, include_info: bool = True, dest_path: Path):
```
Comment (henryiii) [inline bin/projects.py:168]:
```suggestion
def render_projects(projects: list[Project], *, dest_path: Path, include_info: bool = True):
```

Optional arguments last reads better, IMO.

**Review by henryiii — APPROVED** [review]

**Comment by joerick** [comment]
Thanks @henryiii 👍

**Pull request merged**